internal/services/apimanagement/api_management_resource_test.go

```diff
@@ -2059,6 +2124,30 @@ resource "azurerm_resource_group" "test" {
 `, data.RandomInteger, data.Locations.Primary)
 }
 
+func (ApiManagementResource) removeSamples(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
```

Comment:
(as above) if we need this toggle it should live in the Features block - but is there a reason _not_ to remove these by default?

Reply:
Per discussion I've dropped the `remove_samples_on_create` feature flag and made this into a default behavior for v3.0 - the samples are highly unlikely to be retained by someone in a production env and they should be easy to tailor and reconstruct with the corresponding provider resources.